### DISCUSSION TITLE
Intended composition offset behaviour

### DISCUSSION THREAD

**Discussion by thexamlguy** [AvaloniaUI/Avalonia#14827] [Q&A]
Hey all. I am using composition to set the offset of a composition visual with a fixed width but with the height set to the bounding parent using vertical alignment stretch.

On window resized, the offset gets reset back to its original position. I have tried to make it so that the offset is set back to where I'd like it to be on size changed, but that doesn't work.
 
I don't know if that is it intended behaviour though, and would like to understand if there is a better approach? Let's preface that I would like to use composition for this case too.

I have attached a video which demonstrates the behaviour. 

https://github.com/AvaloniaUI/Avalonia/assets/14894973/b332ba20-8b4a-4f91-a188-09336c1aeb12

*edit*

Sample of the issue that I am facing
[AvaloniaApplication3.zip](https://github.com/AvaloniaUI/Avalonia/files/14507795/AvaloniaApplication3.zip)

I have attached a sam

**Comment by timunie** [comment]
> I of course can send a repro sample too.

could be useful.

**Reply by thexamlguy** [reply]
Hi Tim, thank you! I have updated my original post to include a sample project.
